ChocolateyForBusinessQuickStartGuide.md

```diff
@@ -0,0 +1,220 @@
+1. From an Administrator PowerShell Window, browse to the folder where you copied the above two files. Run the following command to install the `chocolatey-c4b-quickstart` package, making sure to pass the **full file path** to your license file:
+
+    ```powershell
+    choco install chocolatey-c4b-quickstart --source . -y --params "'/LicenseFile:<FULL_PATH_TO_LICENSE_FILE>'"
+    ```
+
+    <!--remove {.list-style-type-disc} remove-->
+    > :memo: **NOTE**: We have built this package to SIMPLIFY the initial setup greatly; it will perform the following:
+    > * Download appropriate ".nupkg" files needed to setup Chocolatey.
+    > * Install the license file in the correct directory
+    > * Download a script for Offline install of Chocolatey on endpoints
+1. From an Administrator PowerShell Window, run the following command to install the `chocolatey.extension` package:
+
+    ```powershell
+    choco install chocolatey.extension --source="'C:\choco-setup\packages'"
+    ```
+
+### Repository Server Setup
+
+As recommended, we will assume you have access to the internet from this server. You will need at least a temporary firewall allowance for this, in order to follow the below steps.
+
+1. Copy the entire `C:\choco-setup` directory from the Administrator workstation to the same location on your repository server.
+1. From an Administrator PowerShell Window, run the following command to install Chocolatey:
+
+    ```powershell
+    Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
+    ```
+1. From an Administrator PowerShell Window, run the following command to copy the license into the correct directory:
+
+    ```powershell
+    $null = New-Item $env:ChocolateyInstall\license -ItemType Directory -Force
+    Get-ChildItem -Path "C:\choco-setup" -Recurse | Where-Object { $_.Name -match "'chocolatey.license'" } | Copy-Item "'$($_.Fullname)'" "'$env:ChocolateyInstall\chocolatey.license.xml'" -Force
+    ```
+1. From an Administrator PowerShell Window, run the following command to install the `chocolatey.extension` package:
+
+    ```powershell
+    choco install chocolatey.extension --source="'C:\choco-setup\packages'"
+    ```
+1. From an Administrator PowerShell Window, run the following command to install the `nexus-repository` package:
+
+    ```powershell
+    choco install nexus-repository -y --no-progress
+    ```
+    > :warning: **WARNING** 
+    > If you have already installed your Nexus repository, then the above step is not required. This guide assumes you are starting form scratch. It may be easier, if you are following this verbatim and don't have anything in your repository yet, to simply run `choco uninstall nexus-repository -y` and then `choco install nexus-repository -y`. As the next step requires a freshly-installed Nexus repository, with admin passwords not reset, this would be the most compatible approach.

```

Comment:
We typically add a space between warning and the text

> :warning: **WARNING**
>
> If you ....

Reply:
Updated just now